src/webpack/loaders/devSiteEntry.js

```diff
@@ -0,0 +1,83 @@
+const lodashTemplate = require('lodash.template');
+const { getOptions } = require('loader-utils');
+
+const generateNavigationConfig = require('../loaderUtils/generateNavigationConfig');
+
+/**
+ * Generate the example with the supplied file.
+ * Don't use an arrow function
+ */
+const loader = async function loader(template) {
+  const callback = this.async();
+
+  const imports = {};
+  let index = 0;
+
+  const addImport = (path) => {
+    let ident = imports[path];
+    if (ident) {
+      return ident;
+    }
+    ident = `import${index}`;
```

Comment:
Should this be:
```suggestion
    ident = `import(${index})`;
```

Reply:
What does index represent here? I'm not following exactly what it's representing

Reply:
nah, this is the very generically named import name, so it would render like this

```javascript
 import import0 from 'terra-icon/lib/icon/IconAllergy';
```

Reply:
I'll rename it importName to reduce confusion.